core/util/createBitmaps.js

```diff
@@ -133,17 +127,7 @@ function delegateScenarios (config) {
 
   const asyncCaptureLimit = config.asyncCaptureLimit === 0 ? 1 : config.asyncCaptureLimit || CONCURRENCY_DEFAULT;
 
-  if (/chrom./i.test(config.engine)) {
-    const PORT = (config.startingPort || CHROMY_STARTING_PORT_NUMBER);
-    var getFreePorts = require('./getFreePorts');
-    return getFreePorts(PORT, scenarioViews.length).then(freeports => {
-      console.log('These ports will be used:', JSON.stringify(freeports));
-      scenarioViews.forEach((scenarioView, i) => {
-        scenarioView.assignedPort = freeports[i];
-      });
-      return pMap(scenarioViews, runChromy, { concurrency: asyncCaptureLimit });
-    });
-  } else if (config.engine.startsWith('puppet')) {
+  if (config.engine.startsWith('puppet')) {
```

Comment:
Could we leave the `/chrom./` case and throw and error here -- something like -- _Chromy is no longer supported in version 5+.  Please use version 4.x.x for chromy support._

Reply:
updated!

Reply:
updated!